### DISCUSSION TITLE
milvus-em cannot connect milvus server

### DISCUSSION THREAD

**Discussion by bitxinyu** [milvus-io/milvus#4375] [Q&A and General discussion] [answered]
hello, thanks for your amazing work. 

I start a milvus-em service ,but cannot connect milvus server. 
![image](https://user-images.githubusercontent.com/11964010/101157156-9cc94e00-3664-11eb-9f92-9f34d8531eba.png)
every step was follwed the tutorials. 

1. start milvus by
```
docker run -d --name milvus_cpu_0.11.0 \
-p 19530:19530 \
-p 19121:19121 \
-v /home/$USER/milvus/db:/var/lib/milvus/db \
-v /home/$USER/milvus/conf:/var/lib/milvus/conf \
-v /home/$USER/milvus/logs:/var/lib/milvus/logs \
-v /home/$USER/milvus/wal:/var/lib/milvus/wal \
milvusdb/milvus:0.11.0-cpu-d101620-4c44c0
```
I checked that the service is running，
![image](https://user-images.githubusercontent.com/11964010/101157917-d77fb600-3665-11eb-8ce3-f0d8b38f8cf5.png)

2.  then start milvus-em gui by
```
docker run -d -p 3000:80 -e API_URL=http://0.0.0.0:3000 milvusdb/milvus-em:v0.5.0
```
but cannot connect milvus.

this is my `docker container ls` result:
![image](https://user-images.githubusercontent.com/11964010/101158073-17469d80-3666-11eb-9a83-d0f2c0e766dd.png)

**Comment by shiyu22** [comment, marked as answer]
Hi,
The commands and versions of Milvus and EM that you started seem to be fine.

Please check the status of Milvus first, with `docker logs milvus_cpu_0.11.0`.

**Reply by bitxinyu** [reply]
It seems to be working fine
![image](https://user-images.githubusercontent.com/11964010/101281265-0aae7a80-3809-11eb-99f1-7674b508b07a.png)

**Reply by miia12** [reply]
Are you visiting EM in a local browser? If not, change the host to the corresponding **IP**. if you are visiting locally, change the host to **localhost** or **local IP**.

**Reply by bitxinyu** [reply]
> Are you visiting EM in a local browser? If not, change the host to the corresponding **IP**. if you are visiting locally, change the host to **localhost** or **local IP**.

yes, all services are run locally in my PC. so i use `127.0.0.1` to connect milvus. but I failed.
finally, using corresponding **IP** solve my problem. :grinning: 
This is so weird. I still don't the reason. :confused:

**Reply by miia12** [reply]
Probably because you're using a VPN.

**Reply by bitxinyu** [reply]
e. I have closed VPN before using docker.
thanks anyway :grinning:
